source/connect/connection-targets.txt

```diff
@@ -0,0 +1,107 @@
+.. _c-connection-targets:
+
+==========================
+Choose a Connection Target
+==========================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: connection string, URI, server, settings, client
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to use a connection string and a ``mongoc_client_t`` structure
+to connect to different types of MongoDB deployments.
+
+Atlas
+-----
+
+To connect to a MongoDB deployment on Atlas, include the following elements
+in your connection string:
+
+- The URL of your Atlas cluster
+- Your MongoDB username
+- Your MongoDB password
+
+Then, pass your connection string to the ``mongoc_client_new()`` function.
+
+.. tip::
+
+   Follow the :atlas:`Atlas driver connection guide </driver-connection>`
+   to retrieve your connection string.
+
+When you connect to Atlas, we recommend using the {+stable-api+} client option to avoid
+breaking changes when Atlas upgrades to a new version of {+mdb-server+}.
+
+.. TODO, uncomment this and link to correct Stable API page for C, once written.
+.. To learn more about the {+stable-api+} feature, see the :ref:`<c-stable-api>`
+guide.
+
+The following code shows how to use the {+driver-short+} to connect to an Atlas cluster. The
+code also uses the ``mongoc_server_api_new()`` function to specify a {+stable-api+} version.
+
+.. literalinclude:: /includes/connect/connection-targets-stable-api.c
+   :language: c
+   :start-after: start-connect
+   :end-before: end-connect
+   :dedent:
+
+Local Deployments
+-----------------
+
+To connect to a local MongoDB deployment, use ``localhost`` as the hostname. By
+default, the ``mongod`` process runs on port 27017, though you can customize this for
+your deployment.
+
+The following code shows how to use the {+driver-short+} to connect to a local MongoDB
+deployment:
+
+.. literalinclude:: /includes/connect/connection-targets-local.c
+   :language: c
+   :start-after: start-connect
+   :end-before: end-connect
+   :dedent:
+
+Replica Sets
+------------
+
+To connect to a replica set, specify the hostnames (or IP addresses) and 
+port numbers of the replica-set members.
+
+If you aren't able to provide a full list of hosts in the replica set, you can 
```

Comment:
S: 
```suggestion
If you aren't able to provide a full list of hosts in the replica set, you can 
```
```suggestion
Instead of providing a full list of hosts in the replica set, you can 
```